pyfr/integrators/dual/pseudo/pseudocontrollers.py

```diff
@@ -56,36 +56,141 @@ class DualPseudoNoneController(BaseDualPseudoController):
     pseudo_controller_name = 'none'
 
     @property
-    def _controller_needs_lerrest(self):
+    def _pseudo_controller_needs_lerrest(self):
         return False
 
-    def conv_mon(self, i, minniters):
+    def convmon(self, i, minniters):
+        # Increment the step count
+        self.npseudosteps += 1
+
         if i >= minniters - 1:
             # Subtract the current solution from the previous solution
             self._add(-1.0, self._idxprev, 1.0, self._idxcurr)
 
             # Compute the normalised residual
             resid = tuple(self._resid(self.dtau, self._idxprev))
+
+            self.pseudostepinfo.append((self.npseudosteps, i + 1, resid))
```

Comment:
No need for self.npseudosteps - 1, since the step count increment was  before stepinfo.append even in the previous version.